catalog/spatial/geocoding/spatial-geocoding-offline-catalog/src/main/java/org/codice/ddf/spatial/geocoding/query/GazetteerQueryCatalog.java

```diff
@@ -144,6 +149,39 @@ public GazetteerQueryCatalog(CatalogFramework catalogFramework, FilterBuilder fi
         .collect(Collectors.toList());
   }
 
+  @Override
+  public List<String> getSuggestedNames(String queryString, int maxResults)
+      throws GeoEntryQueryException {
+    Map<String, Serializable> suggestProps = new HashMap<>();
+    suggestProps.put(SUGGESTION_REQUEST_KEY, queryString);
+
+    Query suggestionQuery = new QueryImpl(filterBuilder.attribute(Core.TITLE).text(queryString));
+
+    QueryRequest suggestionRequest = new QueryRequestImpl(suggestionQuery, suggestProps);
+
+    try {
+      QueryResponse suggestionResponse = catalogFramework.query(suggestionRequest);
+
+      Map<String, List<Suggestion>> suggestionMap =
+          (Map<String, List<Suggestion>>)
+              suggestionResponse.getPropertyValue(SUGGESTION_RESULT_KEY);
+      if (suggestionMap != null) {
+        List<Suggestion> suggestions =
+            suggestionMap
+                .values()
+                .stream()
+                .flatMap(List::stream)
+                .limit(maxResults)
+                .collect(Collectors.toList());
```

Comment:
Shouldn't the `suggestions` be returned here? It looks like this method always returns an empty list.

Reply:
This stream deconstructs the Suggestion objects into the name strings we're looking to display and suggest.

Reply:
Yeah but the list isn't used after it is created

Reply:
Hmm, you're right. I think I was going back and forth between returning from the stream and creating an object to return, and ended up doing neither. This PR is kinda stale, so I'll fix this and make sure everything works.